### DISCUSSION TITLE
Idea: Discord community

### DISCUSSION THREAD

**Discussion by lajbel** [replit/kaboom#153] [Ideas]
Would be interesting

**Comment by lajbel** [comment]
I made this little server, if someone wants to ask some kaboom or just talk, they can come in :)

https://discord.gg/3rJ6aVNKs8t

**Comment by slmjkdbtl** [comment]
This link is broken to me? I just made an official one: https://discord.gg/dw2KDbt3

**Reply by GizzmoShifu** [reply]
Link is invalid for me, is there a long-lived link instead of just a short-lived one?

**Reply by lajbel** [reply]
https://discord.gg/aQ6RuQm3TF

**Comment by geocine** [comment]
> This link is broken to me? I just made an official one: https://discord.gg/dw2KDbt3

May you resend invite, invite expired

**Reply by lajbel** [reply]
https://discord.com/invite/aQ6RuQm3TF
